packages/app/features/Send/PasscodeInput/index.tsx

```diff
@@ -52,19 +54,22 @@ const PasscodeInput: FC<Props> = ({ navigator, item, activatedId }) => {
 			const payload: TransactionPayload = {
 				sender: sender,
 				receiver: receiver,
-			} as TransactionPayload;
+				tokenForFee: token as Token,
+			} as unknown as TransactionPayload;
```

Comment:
please explain this `as unknown as TransactionPayload` type

Reply:
eslint fix `as TransactionPayload` to `as unknown as TransactionPayload`